blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitContent.java

```diff
@@ -0,0 +1,192 @@
+package io.jenkins.blueocean.blueocean_git_pipeline;
+
+import io.jenkins.blueocean.commons.ErrorMessage;
+import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmContent;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.export.Exported;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vivek Pandey
+ */
+public class GitContent extends ScmContent {
```

Comment:
You're probably right, I'll make sure there's not duplicated stuff unless it's necessary